woocommerce/woocommerce
# Additional System Status Options

### All Submissions:






### Changes proposed in this Pull Request:
This PR adds three new options to the System Status page. It adds an option under settings to show if the site is connected to WooCommerce.com, useful for troubleshooting issues relating to premium extension updates and licenses, and then also adds two new sections that shows Dropin Plugins and Must Use Plugins, both these sections will be hidden by default if nothing exists there.

![](https://cld.wthms.co/mWuodx+)
![](https://cld.wthms.co/yZhyCR+)

### How to test the changes in this Pull Request:

1. Connect your site to WooCommerce.com via WooCommerce -> Extensions -> WooCommerce.com Subscriptions
2. Install Dropin plugins as well as Must Use plugins
3. Visit the System Status report and ensure all your plugins are listed in the new sections
4. Check the correct status of your WooCommerce.com account connection is shown.

### Other information:




### Changelog entry

> Add - MU, Dropin plugins and WooCommerce.com account connection status to System Status Report.
